atlp-rwanda/script-heroes-bn-backend
# #174797511-Swagger Documentation

#### What does this PR do?
Configure swagger for API documentation
#### Description of Task to be completed?
- Creating swagger.json
- Installing swagger-UI-express
- Configure end point for documentation
#### How should this be manually tested?
- Clone this repo: `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch working directory: `cd script-heroes-bn-backend`
- Checkout the branch: `git checkout  ch-setting-swagger-doc-#174797511`
- Install all dependencies : `npm install`
- Run your server:`node src/index.js`
- Then head to your browser and type in: `http://localhost:3000/api-docs`
> You should see documentation for endpoint.
#### What are the relevant pivotal tracker stories?
[#174797511](https://www.pivotaltracker.com/story/show/174797511)
#### Screenshots (if appropriate)
<img width="1418" alt="Screen Shot 2020-09-18 at 13 17 55" src="https://user-images.githubusercontent.com/51243753/93591879-75da7480-f9b1-11ea-8090-46fefe6a7b1c.png">
